### ISSUE TITLE
Reply to a sent message that has not yet been responded to

### ISSUE THREAD

**Issue by jtwiest** [dacastro4/laravel-gmail#92]
I'm trying to figure out how to reply to messages that has not yet been responded to, the use case is sending a follow-up email a few days after the original one went out.

At the most basic level I'd expect the following to work
```
        $message = LaravelGmail::message()->preload()->get( 'some_id' );
        $message->message('my response body')->reply();
```
However I'm getting a `Recipient Address Requred` error which I can fix by adding `->to('someemail')` before the `reply()`

Question 1: I'm confused why I need to supply a to email since I'd expect the existing message to already have that information and be able to use it. 

Question 2: Even if I do supply this value, it's not adding the message to the thread but instead starting a new one for the recipient and I haven't been able to figure this part out yet.

I did some digging and found that the 'In-Reply-To' header as well as the 'Subject' headers need to match the existing email however I'm not seeing those values being made available when they're referenced here: https://github.com/dacastro4/laravel-gmail/blob/8007c9286b78e4376c63182a7e53d6928ee6158c/src/Traits/Replyable.php#L308

I can supply that value manually when originally sending the email however that doesn't seem to have any effect.

Love the plugin, thanks for all the hard work!

**Comment by jtwiest** [comment]
Of course, I figure it out a few minutes after I ask the question. The answer was to set the 'In-Reply-To' header and the 'References' header manually before you send the original email.

```
        $mail->setHeader('In-Reply-To', '<email@example.com>');
        $mail->setHeader('References', '<email@example.com>');
```

I did however still need to also supply the `->to()` param when sending the email which in an ideal world should be auto supplied in my mind but that's a very solvable solution as it stands.

**Comment by dacastro4** [comment]
@jtwiest sorry for the delay. Cool that you were able to figure it out and you're correct about the reply and supplying the `->to()`, I'm going to look into that. Thank you

**Comment by jtwiest** [comment]
I'm working on an app that needs to ability to send emails using a cronjob so I'm in the midst of a fork that'll allow you to authenticate users on a request by request basis. Basically taking the mutlipleusercredentails flag a few more steps. I'll post a link to my fork here once I get this updated but I do have it working.

**Comment by jtwiest** [comment]
@dacastro4 Here's the PR for the dynamic auth feature, I still got a ways to go on the project I'm working on so chances are there will be some more changes but wanted to put in on your radar: https://github.com/dacastro4/laravel-gmail/pull/93

**Comment by dacastro4** [comment]
fixed in #87

**Comment by phuclh** [comment]
> I'm trying to figure out how to reply to messages that has not yet been responded to, the use case is sending a follow-up email a few days after the original one went out.
> 
> At the most basic level I'd expect the following to work
> 
> ```
>         $message = LaravelGmail::message()->preload()->get( 'some_id' );
>         $message->message('my response body')->reply();
> ```
> 
> However I'm getting a `Recipient Address Requred` error which I can fix by adding `->to('someemail')` before the `reply()`
> 
> Question 1: I'm confused why I need to supply a to email since I'd expect the existing message to already have that information and be able to use it.
> 
> Question 2: Even if I do supply this value, it's not adding the message to the thread but instead starting a new one for the recipient and I haven't been able to figure this part out yet.
> 
> I did some digging and found that the 'In-Reply-To' header as well as the 'Subject' headers need to match the existing email however I'm not seeing those values being made available when they're referenced here:
> 
> https://github.com/dacastro4/laravel-gmail/blob/8007c9286b78e4376c63182a7e53d6928ee6158c/src/Traits/Replyable.php#L308
> 
> I can supply that value manually when originally sending the email however that doesn't seem to have any effect.
> 
> Love the plugin, thanks for all the hard work!

@jtwiest How did you check if a message that has not yet been responded to or not? Thank you!
